.github/workflows/test.yml

```diff
@@ -16,6 +16,7 @@ jobs:
       - uses: actions/setup-go@v3
         with:
           go-version: '1.17'
```

Comment:
Is `go-version` required if we use `check-latest`?

Reply:
Yes; it is "checking for the latest" on Go 1.17. So, in order to find the latest patch version, we need that option. We shouldn't be consuming base Go `1.17`

Reply:
I guess my question is why do we want to use "the latest on Go 1.17", instead of "the latest Go", which is 1.18 ATM.

Reply:
That's fair - I'm up for that. Let me look into it quick!